lib/emoji.rb

```diff
@@ -104,12 +112,12 @@ def parse_data_file
               append_unicode.call(emoji, "#{raw}#{VARIATION_SELECTOR_16}")
             end
           end
-          raw_emoji.fetch(:tags).each { |tag| emoji.add_tag(tag) }
+          raw_emoji.fetch(:tags).each { |tag| emoji.add_tag(dedup.call(tag)) }
 
-          emoji.category = raw_emoji[:category]
-          emoji.description = raw_emoji[:description]
-          emoji.unicode_version = raw_emoji[:unicode_version]
-          emoji.ios_version = raw_emoji[:ios_version]
+          emoji.category = dedup.call(raw_emoji[:category])
+          emoji.description = dedup.call(raw_emoji[:description])
```

Comment:
Each description is unique. Does this really benefit from deduping?

Reply:
This dedup _does_ result in fewer objects (not that much fewer, about 400). It looks like description and aliases can sometimes share the same string.